docs/content/docs/trame_v2_intro.md

```diff
@@ -18,4 +18,4 @@ With trame v2, we are deprecating PyWebVue in favor of our trame suite (trame +
 
 Also now that we have formalized how new Python modules should interface with trame. The community can more simply extend trame with their new widgets and/or UI elements.
 
-Now you might be wondering if a trame v3 is planed with yet another breaking change in another 6 months? Well to be honest we may have released too early pywebvue and trame but they were so game changer that it was hard to keep them for ourself. To be honest, I don't see a v3 anytime soon as v2 has gave us a chance to review our api, fixed what we didn't like and provide an easy path across our various use-cases.
\ No newline at end of file
+Now you might be wondering if a trame v3 is planed with yet another breaking change in another 6 months? Well to be honest we may have released too early pywebvue and trame but they were so game changer that it was hard to keep them for ourself. To be honest, I don't see a v3 anytime soon as v2 has gave us a chance to review our api, fixed what we didn't like and provide an easy path across our various use-cases.
```

Comment:
I didn't change anything about this line. I did the above in the in-browser editor, so I don't know if it trimmed/added a trailing new-line to the file.